### ISSUE TITLE
when zoomed out of BAM tracks, click to expand and show summary

### ISSUE THREAD

**Issue by nathandunn** [GMOD/Apollo#796]

**Comment by monicacecilia** [comment]
- Basically, turn that ugly JSON _Edit Config_ into an interface that is easier to use (likely clickable).
Hybrid viz: histogram when zoomed out, single reads as you get closer. -- and you can configure when the switch occurs (i.e. how 'zoomed in' do you have to be in order to see the reads.
- Remember that we also want to add the ability to visualize strandedness for each read.

**Comment by nathandunn** [comment]
@monicacecilia  As much as I like this, I think this is pretty involved and likely beyond the scope of 2.0.5 (and maybe better handled partially JBrowse).    Let me know if you think I'm wrong.

**Comment by childers** [comment]
Is this issue related to the summary histogram feature in JBrowse?  I was trying to set summary histograms for BAM files in Apollo 2.1.0 with no success.  If this is different, I'm happy to open this as a new issue.

For reference:
http://gmod.org/wiki/JBrowse_Configuration_Guide#Configuring_Summary_Histograms

Here is an example track stanza:
```shell
      {
         "histograms.urlTemplate" : "bigwig/RNASeq.bw",
         "storeClass" : "JBrowse/Store/SeqFeature/BAM",
         "urlTemplate" : "bam/RNASeq.bam",
         "histograms.storeClass" : "JBrowse/Store/SeqFeature/BigWig",
         "type" : "JBrowse/View/Track/Alignments2",
         "label" : "RNASeq",
         "key" : "RNASeq"
      },
``` 
The view never switches over to the histogram view.  Instead it just fails over to the normal "Too much data to show..." message.

**Comment by nathandunn** [comment]
Yeah, this is definitely another issue.    This shouldn't have changed between Apollo 1 and 2, but who knows.  We also updated JBrowse, as well, so it might be worth testing it on different versions of JBrowse, as well.

**Comment by childers** [comment]
Thanks for clearing that up. FYI, it is working, I just had a formatting issue in my JSON.

**Comment by nathandunn** [comment]
ahaa!
